NFIQ2/NFIQ2Algorithm/include/nfiq2_algorithm.hpp

```diff
@@ -11,54 +11,88 @@
 #include <unordered_map>
 
 namespace NFIQ2 {
-/** Wrapper to return quality scores for a fingerprint image */
+
+/**
+ * 	This class serves as a wrapper to return quality scores
+ * 	for a fingerprint image.
+ */
```

Comment:
Wrapper for applying the trained random forest parameters to quality features to compute an NFIQ2 score